test/config.spec.ts

```diff
@@ -148,6 +148,7 @@ describe('Test config defaults', () => {
       expect.objectContaining({
         sellToken: expect.any(String),
         receiveToken: expect.any(String),
+        network: expect.any(Object),
```

Comment:
Would be nice to check `network`, like
```ts
network.forEach(n => expect.objectContaining(...)
```

Reply:
added but it takes a long time on this test